images/cilium-test/cilium-test-gke.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+nodes() {
+  kubectl get nodes --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}'
+}
+
+until [ "$(nodes | wc -l)" -eq  2 ] ; do sleep 5 ; done
+
+index=1
+for node in $(nodes) ; do
+  kubectl label --overwrite node "${node}" "cilium.io/ci-node=k8s${index}"
+  index=$((index+1))
+done
+
+kubectl create ns cilium || true
+
+CILIUM_IMAGE="${1}"
+CILIUM_OPERATOR_IMAGE="${2}"
+
+export CILIUM_IMAGE CILIUM_OPERATOR_IMAGE
```

Comment:
Nit: These lines can be consolidated:

```suggestion
export CILIUM_IMAGE="${1}"
export CILIUM_OPERATOR_IMAGE="${2}"
```

There are a few other occurrences of this.